buildSrc/src/main/kotlin/Deps.kt

```diff
@@ -68,10 +68,17 @@ object Deps {
                 iosX64 = "dev.icerock.moko:parcelize-iosx64:${Versions.Libs.MultiPlatform.mokoParcelize}",
                 iosArm64 = "dev.icerock.moko:parcelize-iosarm64:${Versions.Libs.MultiPlatform.mokoParcelize}"
             )
+            val mokoGraphics = MultiPlatformLibrary(
+                common = "dev.icerock.moko:graphics:${Versions.Libs.MultiPlatform.mokoGraphics}",
+                iosX64 = "dev.icerock.moko:graphics-iosx64:${Versions.Libs.MultiPlatform.mokoGraphics}",
+                iosArm64 = "dev.icerock.moko:graphics-iosarm64:${Versions.Libs.MultiPlatform.mokoGraphics}"
+            )
+
         }
 
         object Jvm {
             const val kotlinPoet = "com.squareup:kotlinpoet:${Versions.Libs.Jvm.kotlinPoet}"
+            const val koson = "com.lectra:koson:${Versions.Libs.Jvm.koson}"
```

Comment:
why you not use already known kotlin-serialization? for what this new library? :(

Reply:
i think we should use kotlinx-serialization

Reply:
Migrated to kotlinx-serialization